code/modules/food_and_drinks/recipes/drinks_recipes.dm

```diff
@@ -633,6 +633,20 @@
 	required_catalysts = list("enzyme" = 5)
 	mix_message = "The rice grains ferment into a clear, sweet-smelling liquid."
 
+/datum/chemical_reaction/peppermint_patty
+	name = "Peppermint Patty"
+	id = "peppermint_patty"
+	results = list("peppermint_patty" = 10)
+	required_reagents = list("hot_coco" = 6, "creme_de_cacao" = 1, "creme_de_menthe" = 1, "vodka" = 1, "menthol" = 1)
+	mix_message = "The coco turns mint green just as the strong scent hits your nose."
```

Comment:
cocoa not coco

Reply:
/datum/chemical_reaction/hot_coco
--
49 | name = "Hot Coco"
50 | id = "hot_coco"
51 | results = list("hot_coco" = 5)
52 | required_reagents = list("water" = 5, "cocoa" = 1)

Reply:
![image](https://user-images.githubusercontent.com/33264221/41305470-d88e2bd4-6e27-11e8-84d5-5cec187486a3.png)